llvm/utils/lit/lit/llvm/config.py

```diff
@@ -131,6 +131,8 @@ def __init__(self, lit_config, config):
             features.add("msan")
         if "undefined" in sanitizers:
             features.add("ubsan")
+        if "thread" in sanitizers:
```

Comment:
How much are these features used? I think the naming is somewhat ambiguous: it could mean "LLVM built with sanitizer X" (which is the meaning of that flag) but I could also read it as "default to building with sanitizer x enabled" or even "<sanitizer> is supported on the current target". Maybe an explicit `built-with-<sanitizer>` or `llvm-instrumented-with-<sanitizer>` would be better?

Reply:
looks like ~200 uses today. I think unless we renamed all of them people would be surprised if tsan was an outlier here

Reply:
Agreed that it doesn't make sense to rename just one of them. And 200 is quite a bit of churn so I'm fine just keeping them as is.